### DISCUSSION TITLE
Make nemo 'Connect to Server ...' with ownCloud more user friendly

### DISCUSSION THREAD

**Discussion by jnweiger** [linuxmint/discussions#68] [Ideas]
Nemo can mount webdav shares with its 'Connect to Server ...' menu.
The for asks for hostname, protocol (webdav via HTTP or HTTPS), username and password.
These are obviously needed.
It also asks for Folder: [/] which is somewhat problematic, as it is not (only) a folder inside the user's home, but needs to have a technical route prefix that is specific to webdav.
In the example of ownCloud, that prefix is `/remote.php/dav/files/`USERNAME`/`

With the default foder `/` the attempt is rejected with a not found error:
![grafik](https://github.com/linuxmint/discussions/assets/1108546/f391f0b3-8d04-4f3d-bf2a-6c3c827960d5)

Is it possible to autodetect or suggest the correct folder prefix, so that the user does not need to remember this technical route prefix?
We could e.g. check https://HOSTNAME/version.php and see that we are connecting to an ownCloud webdav server.

**Comment by kacperpaczos** [comment]
Why you don't use online accounts from gnome?
